app/components/ListItem/index.js

```diff
@@ -0,0 +1,104 @@
+/* @flow */
+
+import React from 'react';
+
+type Props = {
+  content: string,
+  id: any,
+  updateItem: () => void,
+  removeItem: () => void,
+};
+
+type State = {
+  editStatus: boolean,
+  content: string,
+  id: any,
+};
+
+export default class ListItem extends React.Component {
+
+  constructor(props: Props) { // eslint-disable-line no-useless-constructor
+    super(props);
+    this.state = {
+      editStatus: false,
+      content: props.content,
+      id: props.id,
+    };
+
+    (this:any).handleRemove = this.handleRemove.bind(this);
+    (this:any).openEditMode = this.openEditMode.bind(this);
+    (this:any).closeEditMode = this.closeEditMode.bind(this);
+    (this:any).handleChange = this.handleChange.bind(this);
+    (this:any).renderEditMode = this.renderEditMode.bind(this);
+    (this:any).renderViewMode = this.renderViewMode.bind(this);
+  }
+
+  state: State;
+
+  props: Props;
+
+  handleRemove() {
+    this.props.removeItem(this.state.id);
```

Comment:
since this is a one-line function, could we just have this inline with the jsx?
like `onClick={() => this.props.removeItem(this.state.id)}`